.github/workflows/pytest.yml

```diff
@@ -160,6 +161,7 @@ jobs:
         os: [windows-latest]
         floatx: [float32, float64]
         test-subset:
+          - pymc/tests/test_variational_inference.py
```

Comment:
This will create a separate new job for windows. Instead, you can add it to any of the jobs below, which are all pretty short anyway

Reply:
I fixed the points above